packages/client-adapters/src/adapters/logeion/adapter.js

```diff
@@ -0,0 +1,46 @@
+import DefaultConfig from '@clAdapters/adapters/logeion/config.json'
+import BaseAdapter from '@clAdapters/adapters/base-adapter'
+
+class AlpheiosLogeionAdapter extends BaseAdapter {
+  /**
+   * Adapter uploads config data
+   * @param {Object} config - properties with higher priority
+  */
+  constructor (config = {}) {
+    super()
+    this.config = this.uploadConfig(config, DefaultConfig)
+    this.available = this.config.availableLangs.includes(this.config.lang)
+  }
+
+  /**
+  * This method retrieves a list of words for lookup autocomplete
+  * @param {String} text - text for retrieving variants
+  * @return {Array} - array of words
+  */
+  async getWords (text) {
+    try {
+      const url = this.createFetchURL(text)
+
+      const wordsVariants = await this.fetch(url)
+
+      if (wordsVariants.words && Array.isArray(wordsVariants.words)) {
+        return wordsVariants.words.slice(0, this.config.limit)

```

Comment:
I think we should go with your suggestion of filtering the results by character to include only the requested language. See additional comments on this in the PR conversation thread.